.github/workflows/node.js.yml

```diff
@@ -21,6 +21,13 @@ jobs:
             VSCODE_TEST_VERSION: ${{ matrix.vscode-version }}
             NODE_OPTIONS: '--max-old-space-size=8192'
         steps:
+            # Setup for git-secrets
+            - name: Setup git-secrets executable
+              run: |
+                  mkdir $HOME/bin
+                  curl -o $HOME/bin/git-secrets https://raw.githubusercontent.com/awslabs/git-secrets/99d01d58ebcc06e237c0e3f3ff5ae628aeef6aa6/git-secrets
+                  chmod +x $HOME/bin/git-secrets
```

Comment:
not in $PATH.

But anyway, since we have a test file dedicate for this, let's move the `curl` logic into the test file. That avoids the ambiguity related to $PATH (which could result in different git-secrets versions if run locally, for example)